pkg/build/api/types.go

```diff
@@ -562,6 +562,10 @@ type DockerBuildStrategy struct {
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
 
+	// Args contains any build arguments that are to be passed to Docker.  See
```

Comment:
Env is used already, I don't think we could consolidate BuildArgs and Env together